consoles/s3270.pm

```diff
@@ -246,7 +246,10 @@ sub expect_3270() {
                     push @$result, $line;
                 }
 
-                confess "status line matches neither buffer_ready nor buffer_full:\n" . Dumper($result) . $status_line;
+                # TODO the following should be "confess" but since
+                # 7a2e1b41 confess itself dies inside the die handler w/o
+                # helpful output
+                warn "status line matches neither buffer_ready nor buffer_full:\n" . Dumper($result) . $status_line;
```

Comment:
yes as it looks like we can handle this better from a higher layer. The confess was visible some time ago but since 7a2e1b41 it seems to be swallowed by a die handler somewhere else so the backtrace was never visible and the test would just die and we don't have a chance to handle it. I think https://openqa.suse.de/tests/437837/file/autoinst-log.txt is one of these cases, couldn't find a more obvious example right now.